src/mds/mdstypes.cc

```diff
@@ -285,12 +286,15 @@ void inode_t::encode(bufferlist &bl, uint64_t features) const
   ::encode(last_scrub_version, bl);
   ::encode(last_scrub_stamp, bl);
 
+  if (features & CEPH_FEATURE_FS_BTIME)
+    ::encode(btime, bl);
```

Comment:
I'd have to check the call chain to figure out what features we pass down when. Certainly an OSD will never be looking at these structures directly, but we encode them when writing out to omaps for storage on an OSD.